README.md

```diff
@@ -206,20 +205,13 @@ Show your support by giving us a star :star:, to help others discover `genezio`
 
 There are a growing number of awesome projects deployed with `genezio` and we want to shout out about them.
 
-If you deployed a project using `genezio` let us know on [Discord](https://discord.gg/uc9H5YKjXv) and we will add it to our [Hall Of Fame](https://github.com/Genez-io/genezio#hall-of-fame).
+If you deployed a project using `genezio` let us know on [Discord](https://discord.gg/uc9H5YKjXv).
 
-# Community projects
+# Tutorials
 
-Below you can find projects build by the community and deployed with `genezio`.
+Below you can find projects and tutorials for building and deploying on Genezio: 
```

Comment:
Maybe add the tutorial link directly here instead of creating a list with a single item

Reply:
Also there do not seem to be any projects listed?